### ISSUE TITLE
.playerbot bot self

### ISSUE THREAD

**Issue by LitchLight** [liyunfan1223/mod-playerbots#249]
Love watching my toon in self bot mode. Have noticed that it is possible to /whisper to my bot when he is running. eg. /whisper Botname grind

Is there a way to persuade the bot to STOP?! 
Currently have had to log out each time my toon needs help!

Also would be nice if they followed clicks made by the player while out of combat. 
Eg. right-click to force interatcion with a gameobject?

Also - noticed the "go" command. 
Not so useful. Would be better if "go Undercity" (for example) worked.

Is there a list of commands have not been able to work it out from the sourcecode.
Could someone post these?

xxx.
LL.

**Comment by azcguy** [comment]
Here are some commands https://github.com/whipowill/wow-addon-playerbots/blob/master/MANUAL.md
Not all work, use item on item doesnt work, maybe it did at some point.

**Comment by LitchLight** [comment]
Very kind - thank you.

Noticed the following command missing from that linked above ...
.playerbots bot self

Also - once in this mode - how to stop the bot - to exit "bot self" without relog? 
Or is that the only way?

**Comment by fuzzdeveloper** [comment]
This documentation might be useful too (its for mangos bot, but some of the commands still work):
https://ike3.github.io/mangosbot-docs/doc/playerbot/Commands.html

I've also made a bit of a guide myself which includes some of the commands I used for doing vanilla raids with the bots (also has notes of my experience using them for different raids):
https://docs.google.com/document/d/1OhVMt2CtCTDMAzgit0AO2j51pkdk_WGWZg55-6HEOx0

Honestly the best documentation you're likely to find for the bot is the sourcecode itself, because it's changed so much since any documentation was written (it's how I've figured out how to use many of the commands). Much can be learned by looking up the spoken commands defined here and then jumping to the code that processes each command:
https://github.com/liyunfan1223/mod-playerbots/blob/master/src/strategy/actions/ChatActionContext.h

I've also learned a bit from looking at the lua code for wow client addons that control the bots (addons like azcguy linked above), though those have the same problem as documentation (ie: the bot has changed significantly since addon was made).

Something else worth mentioning (not covered in links above) are these commands that you you enter it directly into worldserver console:
- playerbot rndbot init - basically re-rolls all the random bots, this is useful after changing some of the config settings (eg: min/max levels, probability of certain specs - speaking of which making every warrior a tank and every priest a healer will make life easier for you). Also useful if you've left server running and all bots have out levelled you (these days I've just turned off experience gain for them in the config)
- playerbot rndbot stats - shows statistics (eg: distribution of bots per level/race/faction)
